screens/AppNavigator/screens/TransactionsScreen/TransactionsScreen.tsx

```diff
@@ -1,36 +1,149 @@
+    const TOKEN_NAME: { [key in number]: string } = {
+      0: 'DFI',
+      1: 'tBTC'
+    }
+
+    const tokenId = TOKEN_NAME[row.item.tokenId as number]
+    let amount = new BigNumber(row.item.value)
+
+    if (isPositive) {
+      color = 'green'
+      // TODO(@ivan-zynesis): Simplified, more complicated token transaction should have different icon and desc
+      iconName = 'arrow-down'
+      desc = 'Received'
+    } else {
+      color = 'gray'
+      iconName = 'arrow-up'
+      desc = 'Sent'
+      amount = amount.negated()
+    }
+
+    const option = {
+      id: row.item.id,
+      desc,
+      iconName,
+      color,
+      amount: amount.toFixed(),
+      block: row.item.block.height,
+      token: tokenId
+    }
+
+    const txIcon = <Ionicons name='arrow-down' size={24} color={color} />
+    const amountText = <NumberText value={option.amount} style={{ color }} />
+
+    return (
+      <View key={row.item.id} style={tailwind('bg-white p-2 border-b border-gray-200 flex-row items-center w-full h-16')}>
+        <View style={tailwind('w-full flex-row flex-initial')}>
+          <View style={tailwind('w-8 justify-center')}>
+            {txIcon}
+          </View>
+          <View style={tailwind('flex-auto flex-row justify-center items-center')}>
+            <View style={tailwind('flex-auto flex-col ml-3 justify-center')}>
+              <Text style={tailwind('font-medium')}>
+                {translate('screens/TransactionsScreen', option.desc)}
+              </Text>
+              <Text style={tailwind('font-medium')}>
+                {translate('screens/TransactionsScreen', 'block')}: {option.block}
+              </Text>
+            </View>
+            <View style={tailwind('flex-row ml-3 items-center')}>
+              {amountText}
+              <View style={tailwind('w-16 ml-3 items-start')}>
+                <Text style={tailwind('flex-shrink font-medium')}>
+                  {option.token}
+                </Text>
+              </View>
+            </View>
+          </View>
+          <View style={tailwind('w-8 flex-grow-0 justify-center flex-end')}>
+            <Ionicons name='chevron-forward' size={24} color='gray' />
+          </View>
+        </View>
+      </View>
+    )
+  }
 
-const SettingsStack = createStackNavigator<TransactionsParamList>()
+  const LoadMore = (): JSX.Element|null => {
+    if (hasNext !== true) {
+      return null
+    }
+    return (
+      <View style={tailwind('flex-1 items-center justify-center w-full')}>
+        <Button title={translate('screens/TransactionsScreen', 'load more')} onPress={() => loadData(false)} />
+      </View>
+    )
+  }
 
-export function TransactionsNavigator (): JSX.Element {
   return (
-    <SettingsStack.Navigator>
-      <SettingsStack.Screen
-        name='transactions'
-        component={TransactionsScreen}
-        options={{ headerTitle: translate('screens/TransactionsScreen', 'Transactions') }}
+    <ScrollView style={tailwind('w-full')}>
```

Comment:
You can remove the `ScrollView` wrapper. It throws a warning if you nest scroll containers together. 
![image](https://user-images.githubusercontent.com/25013382/122527587-08499a80-d04e-11eb-995b-5e8a6f828756.png)